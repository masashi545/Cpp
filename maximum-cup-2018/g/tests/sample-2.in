?([?[(?)o]?(?([(2?o]??[?3)o][?4?x??x])?]??5)x??x???][????])x?[??)o?
7
1 2 1 0 0 1 1
