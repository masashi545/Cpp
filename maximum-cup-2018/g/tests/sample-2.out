[([([(1)o][([([(2)o][([(3)o][(4)x])x])o][(5)x])x])x][(6)o])x][(7)o]
